src/model.ts

```diff
@@ -154,18 +154,17 @@ function wrap<A, R>(
   return async function modelWrapped(...args) {
     const { collectionName } = model.collection;
     const { after = [], before = [] } = model.options?.hooks || {};
-    const context = {};
```

Comment:
Please do not remove the `context` variable. We rely on it at Plex to run some custom logic.

If you want to achieve your intended changes with this PR, just attach `result` in the `context` object.